t/t2270-job-dependencies.t

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+
+test_description='Test job dependencies'
+
+. $(dirname $0)/sharness.sh
+
+test_under_flux 4 job
+
+flux setattr log-stderr-level 1
+
+PLUGINPATH=${FLUX_BUILD_DIR}/t/job-manager/plugins/.libs
+
+test_expect_success 'create dep-remove.py' '
+	cat <<-EOF >dep-remove.py
+	import flux
+	from flux.job import JobID
+	import sys
+
+	jobid = flux.job.JobID(sys.argv[1])
+	name = sys.argv[2]
+	topic = "job-manager.dependency-test.remove"
+	print(flux.Flux().rpc(topic, {"id": jobid, "name": name}).get())
```

Comment:
should name be description for consistency? not necessary, sort of borderline, i know this is a test